drivers/OpenMote-CC2538/max44009.c

```diff
@@ -122,15 +119,14 @@ uint16_t max44009_read_light(void)
     i2c_read_byte(MAX44009_ADDRESS, &max44009_data[1]);
 
     exponent = (( max44009_data[0] >> 4 )  & 0x0E);

```

Comment:
If the 4 bits are 0001, then the exponent is 0. In general, if the 4 bits have a decimal representation "x", and x is a odd number, the resulting exponent is x-1. According to the datasheet, this seems not the correct behavior. Maybe we need an if clause to avoid that the resulting exponent is greater than 14, isn't it?